src/components/Tabs/Annotation/AnnotationInputPopover.tsx

```diff
@@ -0,0 +1,169 @@
+import { DeleteOutlined } from "@ant-design/icons";
```

Comment:
![capture](https://github.com/user-attachments/assets/61af24bb-41f3-4e13-82d6-de152c4d5156)

Editable field that appears when clicking on an existing label. Changing the input edits the value in real-time (though this behavior might change if it causes lag elsewhere in the UI).

Clearing the field removes the labels, as does pressing the delete button.